src/ORM/DataList.php

```diff
@@ -1344,36 +1334,30 @@ public function offsetExists($key)
      *
      * The object returned is not cached, unlike {@link DataObject::get_one()}
      *
-     * @param mixed $key
      * @return DataObject
      */
-    #[\ReturnTypeWillChange]
-    public function offsetGet($key)
+    public function offsetGet($key): mixed
```

Comment:
Why haven't you set the type for the argument here like you did for this method in other classes?

Reply:
Fixed.